### ISSUE TITLE
implement sharable links

### ISSUE THREAD

**Issue by DillonRego** [DillonRego/ListTogether#3]

**Comment by AnthonyTennis** [comment]
I think we should focus on other project deliverables. Sharing resources tied to a uuid might get muddy and might be too time consuming given the amount of time we have left

**Comment by laestevez** [comment]
I agree, I wasted a good amount of time trying to figure it out and I couldn't get it to work.
